specs/Terminate Spec.md

```diff
@@ -17,42 +17,54 @@ PowerToys is mainly targeted towards Windows Power Users though it is available
 
-**1.4. Existing Solutions or Expectations**
+- A publicly accpetable name for this PowerToy.
+- The degree to which the process will be terminated.
 
-Users can already acheive the function of this PowerToy by accessing clicking End Task through Task Manager on Windows. Users need to be running Windows 10 and have PowerToys installed to use it.
+**1.4. Existing Solutions or Expectations**
 
+The current methods a user can close a running program in Windows include:
+- Clicking on the close button in the program
+- Closing the program via task manager
+- Using the keyboard shortcut Alt + F4 to close the program
 
 **1.5. Goals/Non-Goals**
 
 - Develop this PowerToy and have sufficient time for testing and integration within our assigned 8 weeks for the project. 
 
-- Come up with a suitable name for this PowerToy, preferrably one accepted by the community.
+- Meet customers expectation with end result of project.
 
 ## 2. Definition of Success
 
 **2.1. Expected Impact: Customer, and Technology Outcomes, Experiments + Measures**
 
-There is currently a sizable number of people who are following the PowerToys project. We hope to meet these people's expectations and receive good feedback from the community when we are done with this PowerToy.
+The PowerToys repo currently has 200+ people watching, over 4000 stars and 109 forks on github despite having an empty repo. Also, this particular PowerToy received a rating of 3.44/5 in the survey we sent out to the community asking them to rate how useful they think it will be. After the PowerToys repo is made public, we hope to see at least a 5% increase in all the Github statistics aforementioned. On completion, a follow up survey will be sent out to the community and an increase in the rating to about 3.75/5 will be a good indicator that we met our customers satisfaction.
 
 ## 3. Requirements
 
 **3.1.	Functional Requirements**
 
-- A keyboard shortcut to display a confirmation window 
-- Cursor morphs when user hovers over the "Yes" option
-- Normal cursor when user hovers over the "No" option 
+Tou use this PowerToy, a user will:
+
+- Click the chosen keyboard shortcut and cursor morphs to indicate that user is in 'Terminate' mode.
+- In Terminate mode, whenever a user clicks on a window, it displays a yes/no prompt for the user to confirm that they want to close said window.
+    - Note: users can configure in their options to disable the yes/no prompt and just terminate apps instantly.
+- Once they are done terminating their apps, user presses keyboard shortcut again to leave terminate mode.
 
 ![Terminate](https://raw.github.com/indierawk2k2/PowerToys-1/images/PT%20Images/Terminate%20Blurred.png "Terminate")
 
 - Keyboard shortcut suggestions
-    - Windows Key + Alt + X
-    - Windows Key + Shift + X 
+    - Alt + Z
+    - Alt + Shift + X 
```

Comment:
@miniksa Are these shortcuts better? If not, do you have any suggestions?

Reply:
Probably. Note that you might be fine using the Windows Key ones, I'm not sure. @bitcrazed would know more about the issues with the Windows Key. I just have a bad feeling about them.

It's a really tricky problem because virtually all combinations are already used by everyone somehow. It might be important to choose one and let people override it somehow. This is why one of the core tenets of the Terminal project is customization.